base/doc/ltnews41.tex

```diff
@@ -743,6 +743,19 @@ \subsection{Fully expand the arguments of \cs{counterwithin} and \cs{counterwith
 \githubissue{1675}
 
 
+\subsection{Ensure that late \cs{write} commands aren't lost}
+
+A non-\cs{immediate} \cs{write} command, issued after the final page
+has been shipped out, is never written because it is waiting for
+another \cs{shipout} to happen. To ensure that such \cs{write}s, for
+example to the \texttt{.aux} file, are nevertheless written, \LaTeX{}
+is now changing its behavior after the final \cs{shipout} and makes
+any further \cs{write} statements automatically \cs{immediate} (i.e.,
+before it executes code in the \hook{enddocument/afterlastpage} hook).
```

Comment:
This needs to be made much more explicit.  

Difficult to make a suggestion due to not following the exact meaning.  

But there should definitely be a new sentence following this:

. . . after the final \shipout.  

Note that this is explained much better next to the code (see comment there).

Reply:
I gave it a try ...